readme/clipper.md

```diff
@@ -43,4 +71,4 @@ Copy and paste the content of both the debugging window and the Firefox console,
 
 # Using the Web Clipper service
 
-The Web Clipper service can be used to create, modify or delete notes, notebooks, tags, etc. from any other application. It exposes an API with a number of methods to manage Joplin's data. For more information about this API and how to use it, please check the [Joplin API documentation](https://joplinapp.org/api/references/rest_api/).
+The Web Clipper service can be used to create, modify or delete notes, notebooks, tags, etc. from any other application. The browser extension form allows you to take notes directly from a webpage instead of having to retype everything. It exposes an API with a number of methods to manage Joplin's data. For more information about this API and how to use it, please check the [Joplin API documentation](https://joplinapp.org/api/references/rest_api/).
```

Comment:
What do you mean the "browser extension form"? Not sure what this sentence means as we don't have this feature